scavenger-collector/src/main/kotlin/com/navercorp/scavenger/util/SqlRayQueryLogEntryCreator.kt

```diff
@@ -28,6 +28,8 @@ class SqlRayQueryLogEntryCreator : QueryLogEntryCreator {
         writeDataSourceName: Boolean,
         writeConnectionId: Boolean
     ): String {
+        val list = listOf<String>()
+        list.iterator()
```

Comment:
This is not being used, please delete it.

Reply:
Thx